swe/options.py

```diff
@@ -14,7 +14,7 @@ class ShallowWaterOptions(Options):
     solve_tracer = Bool(False).tag(config=True)
 
     # Physical
-    bathymetry = FiredrakeScalarExpression(Constant(1.0)).tag(config=True)
+    bathymetry = FiredrakeScalarExpression(None, allow_none=True).tag(config=True)#FiredrakeScalarExpression(Constant(1.0)).tag(config=True)
```

Comment:
Why are you setting default bathymetry to `None`? It's inconsistent with all of my test cases.

Reply:
I changed it as the default in thetis is to initialise it as None. It shouldn't make too much of a difference as presumably you are rewriting your bathymetry anyway and making it a function?

Reply:
Fair enough